src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/views/article.html.twig

```diff
@@ -1,12 +1,51 @@
 {% extends 'base.html.twig' %}
 
 {% block body %}
-    <h2>{{gimme.article.title}}</h2>
-    <p>Id: {{gimme.article.id}}</p>
-    <p>Content: {{gimme.article.body|raw}}</p>
+
+<div id="page" class="container">
+
+ <!-- Content -->
+ <section id="content">
+
+   <div class="row article_content">
+
+{% cache 'article' {gen: article} %}
+
+   	<article class="news_item span8 ">
+			<div class="thumbnail content_text">
+       		<img src="{{ asset('theme/assets/img/images-cms-image-000000103X.jpg') }}" alt="Boat in Copenhagen canals (photo: Source Fabric)">
+       		<h6 class="caption ">Boat in Copenhagen canals</h6>
+    	</div>
+
+	      <div class="content content_text">
+       		<h6 class="info">{{ gimme.article.publishedAt|date("d.m.Y, H:i") }}</h6>
+          {###### We can also generate following timestamps #######}
```

Comment:
as it's demo theme then it can be somehow visible for reader, if not then should be added to docs and removed from here.